### PR TITLE
Report in bulk

### PR DESCRIPTION
**Pull request by jacktang**
We'd like to insert meter data in InfluxDB as one record

### CODE REVIEW

**Comment by surik** [comment]
Hi @jacktang,

thank you for your PR. 

By the way, we already have own bulk write mechanism, see `batch_window_size` option.  Of course, using `exometer_report_bulk` as an additional option is not a bad idea. I will take a look on it.

File: src/exometer_report_influxdb.erl [hunk 1]
```diff
@@ -136,6 +137,34 @@ exometer_report(Metric, DataPoint, _Extra, Value,
...
+    ?info("InfluxDB reporter isn't connected and will reconnect."),
+    {ok, State};
+exometer_report_bulk([{_Metric,[{_DataPoint, _Value}|_]}|_] = Found, _Extra, 
+                     #state{metrics = Metrics} = State) ->
+    {NState, Errors} = 
+        lists:foldl(
+          fun({Metric, DataPoints}, {StateAccIn, ErrorsAccIn}) -> 
+                  case maps:get(Metric, Metrics, not_found) of
+                      {MetricName, Tags} ->
+                          case maybe_send(Metric, MetricName, Tags,
```
Comment (surik) [inline src/exometer_report_influxdb.erl:150]: Did you notice that `maybe_send` has two behaviors depend on `batch_window_size`?
With `batch_window_size=0` each metric from a bulk will be sent in an own request.
With `batch_window_size>0` it isn't sent immediately but goes to a buffer to the further sending in one batch.

Comment (jacktang) [reply src/exometer_report_influxdb.erl:150]: @surik you're right. I missed the `batch_window_size` config. The original code supports the report in bulk.

Comment (surik) [reply src/exometer_report_influxdb.erl:150]: then I will close it

**Pull request closed without merging**